docs/src/modules/components/AppSearch.js

```diff
@@ -1,277 +1,92 @@
 import * as React from 'react';
-import url from 'url';
-import clsx from 'clsx';
-import useLazyCSS from 'docs/src/modules/utils/useLazyCSS';
 import useMediaQuery from '@material-ui/core/useMediaQuery';
 import { makeStyles } from '@material-ui/styles';
 import { alpha, useTheme } from '@material-ui/core/styles';
-import Input from '@material-ui/core/Input';
-import SearchIcon from '@material-ui/icons/Search';
-import { handleEvent } from 'docs/src/modules/components/MarkdownLinks';
-import docsearch from 'docsearch.js';
+import { DocSearch } from '@docsearch/react';
+import '@docsearch/react/dist/style.css';
 import { LANGUAGES_SSR } from 'docs/src/modules/constants';
-import { useUserLanguage, useTranslate } from 'docs/src/modules/utils/i18n';
+import { useUserLanguage } from 'docs/src/modules/utils/i18n';
 
 const useStyles = makeStyles(
   (theme) => ({
     '@global': {
-      '.algolia-autocomplete': {
-        '& .ds-dropdown-menu': {
-          boxShadow: theme.shadows[1],
-          borderRadius: theme.shape.borderRadius,
-          '&::before': {
-            display: 'none',
-          },
-          '& [class^=ds-dataset-]': {
-            border: 0,
-            maxHeight: 'calc(100vh - 100px)',
-            borderRadius: theme.shape.borderRadius,
-            backgroundColor: theme.palette.background.paper,
-          },
+      ':root': {
+        '--docsearch-modal-height': 'calc(100vh - 100px)',
+        '--docsearch-primary-color': theme.palette.primary.main,
+        '--docsearch-text-color': theme.palette.text.primary,
+        '--docsearch-modal-background': theme.palette.background.paper,
+        '--docsearch-searchbox-background': alpha(theme.palette.common.white, 0.15),
+        '--docsearch-searchbox-focus-background': alpha(theme.palette.common.white, 0.25),
+        '--docsearch-key-gradient': alpha(theme.palette.common.white, 0.4),
+        '--docsearch-key-shadow': 'none',
+        '--docsearch-hit-color': theme.palette.primary.main,
+        '--docsearch-hit-shadow': theme.shadows[1],
+        '--docsearch-hit-background': theme.palette.background.paper,
+        '--docsearch-footer-background': theme.palette.background.paper,
+        '--docsearch-footer-shadow': '0 -4px 8px 0 rgba(0, 0, 0, 0.2)',
+        '--docsearch-logo-color': theme.palette.primary.main,
+        '--docsearch-muted-color': 'rgb(127, 132, 151)',
+        '.DocSearch-Form': {
+          '--docsearch-searchbox-shadow': '0 0 0 2px rgba(0, 0, 0, 0.3)',
```

Comment:
@oliviertassinari May be because of global variable in css, but I am able to see it locally, as in screenshots